tests/integration/observer_test/parser_test.go

```diff
@@ -112,8 +111,8 @@ func setupParser(stopChan chan struct{}) parser.Params {
 		Api:                   getMockedBlockAPI(),
 		ParsingBlocksInterval: pollInterval,
 		BacklogCount:          backlogCount,
-		MaxBacklogBlocks:      int64(maxBatchBlocksAmount),
 		TxBatchLimit:          100,
 		StopChannel:           stopChan,
+		BlocksPerRound: 50,
```

Comment:
```suggestion
		BlocksPerRound: 50,
```

Even for test, probably test 10-20 max. there is no need to fetch more